sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/secret_client.hpp

```diff
@@ -43,7 +42,11 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
    * Vault. The client supports creating, retrieving, updating, deleting, purging, backing up,
    * restoring, and listing the secret.
    */
-  class SecretClient final {
+  class SecretClient
```

Comment:
It's changes like these that I wish we weren't reverting. This effort is creating more, redundant work :(

Reply:
It is intentional that this PR is an automated "git revert" - so that we don't stall on conversations, can check it in tonight, and it is easier to prove if I am not introducing new bugs here/not forgetting anything.